sirepo/pkcli/jupyterhublogin.py

```diff
@@ -58,9 +59,25 @@ def maybe_create_sirepo_user(qcall, email, display_name):
         with qcall.auth.logged_in_user_set(u, method=sirepo.auth.METHOD_EMAIL):
             n = sirepo.sim_api.jupyterhublogin.create_user(
                 qcall=qcall,
-                check_dir=True,
             )
             qcall.auth_db.model("UserRole").add_roles(
                 roles=[sirepo.auth_role.for_sim_type("jupyterhublogin")],
             )
         return PKDict(email=email, jupyterhub_user_name=n)
+
+
+def unknown_user_dirs():
+    """Returns directory names for Jupyter users that are not in database.
+    Excludes directories that begin with capital letter.
+    """
+    with sirepo.quest.start() as qcall:
+        m = ""
```

Comment:
list is better. Separate presentation from business logic. Strings are generally problematic, especially with full paths that might name special characters or spaces in them. If our goal is to delete the directories, we can transform with Python or b fixing #293.

Reply:
[3a9f0d1](https://github.com/radiasoft/sirepo/pull/7027/commits/3a9f0d1399801354cea9e77a661893218636e2dd)